pages/accounts/profile.py

```diff
@@ -7,7 +7,6 @@
 
 from pages.accounts import admin, home
 
-
 class Profile(home.AccountsHome):
```

Comment:
This is a PEP8 error; there needs to be 2 blank lines between imports and 1st level elements